Alexandr Antsiferov/lesson7/src/app/pages/Users.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from "react";
+import {fetchUsers} from'../actions/usersAction';
+import { connect } from 'react-redux';
+import UsersList from "../components/UsersList";
+
+class Users extends Component {
+  render() {
+    return (
+      <div>{!this.props.children ? <UsersList users={this.props.users}/> : this.props.children}</div>
```

Comment:
```suggestion
      <UsersList users={this.props.users}/>
```
Компоненту `Users` ничего не передается в качестве `children` :)

Reply:
точно)